equippedgear/src/ts/main.tsx

```diff
@@ -0,0 +1,21 @@
+/**
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+/// <reference path="./main.d.ts"/>
+
+import {events} from 'camelot-unchained';
+import * as React from 'react';
+import * as ReactDOM from 'react-dom';
+import {EquippedGearWindow} from './components/equippedgear-window';
+
+events.on('init', () => {
+  ReactDOM.render(
```

Comment:
Any reason for the long form instead of just:

    ReactDOM.render(<EquippedGearWindow />, document.getElementById('equippedgear'));